File: src/main/java/com/company/payroll/mapper/BankingInfoMapper.java
package com.company.payroll.mapper;

import org.springframework.stereotype.Repository;

import com.company.payroll.model.BankingInfo;

//@Repository
public interface BankingInfoMapper {
    int deleteByPrimaryKey(Integer bId);

    int insert(BankingInfo row);

    int insertSelective(BankingInfo row);

    BankingInfo selectByPrimaryKey(Integer bId);

    int updateByPrimaryKeySelective(BankingInfo row);

    int updateByPrimaryKey(BankingInfo row);
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/mapper/BankingInfoMapper.kt
package com.company.payroll.mapper

import com.company.payroll.model.BankingInfo
import org.springframework.stereotype.Repository

@Repository
interface BankingInfoMapper {
  fun deleteByPrimaryKey(bId: Int): Int
  fun insert(row: BankingInfo): Int
  fun insertSelective(row: BankingInfo): Int
  fun selectList(): List<BankingInfo>
  fun selectByPrimaryKey(bId: Int): BankingInfo?
  fun updateByPrimaryKeySelective(row: BankingInfo): Int
  fun updateByPrimaryKey(row: BankingInfo): Int
}